controller/workitem_blackbox_test.go

```diff
@@ -2895,11 +2893,10 @@ func createSpaceWithDefaults(ctx context.Context, db *gorm.DB) (*space.Space, *i
 		return nil, nil, nil
 	}
 
-	userActive := false
 	itr := &iteration.Iteration{
 		Name:       sp.Name,
 		SpaceID:    sp.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c